docs/source/tagset/elemento-country.rst

```diff
@@ -3,23 +3,22 @@
 <country>
 ^^^^^^^^^
 
-Aparece em
+Aparece em:
+
   :ref:`elemento-aff`
 
-Atributos obrigatórios
-  1. country
- 
-Ocorre
+Atributos obrigatórios:
+
+  1. ``@country``
+
+Ocorre:
+
   Uma vez
 
 
-Identifica o país da afiliação.
- 
-A tag deve possuir o atributo ``@country`` e nele deve ser atribuído o código 
-do país de acordo com a Norma *ISO 3166*, de dois caracteres alfabéticos.
+Identifica o país de afiliação de um autor. O atributo ``@country`` é obrigatório e deve ser preenchido de acordo com a Norma *ISO 3166*, cujos códigos tem dois caracteres alfabéticos em caixa alta.
```

Comment:
Incluído ``elemento-corresp`` em "aparece em" e retirada informação de afiliação e incluída informação adicional que quando for afiliação é obrigatório.